vidur/config/device_sku_config.py

```diff
@@ -0,0 +1,66 @@
+from dataclasses import dataclass, field
+
+from vidur.config.base_poly_config import BasePolyConfig
+from vidur.logger import init_logger
+from vidur.types import DeviceSKUType
+
+logger = init_logger(__name__)
+
+
+@dataclass
+class BaseDeviceSKUConfig(BasePolyConfig):
+    fp16_tflops: int = field(
+        metadata={"help": "The number of TFLOPS the device can achieve in FP16"},
+    )
+    total_memory_gb: int = field(
+        metadata={"help": "The total memory of the device in GB"},
+    )
+
+
+@dataclass
+class A100DeviceSKUConfig(BaseDeviceSKUConfig):
+    fp16_tflops: int = field(
+        default=312,
```

Comment:
i am thinking that it would probably be a little overwhelming for people to see all these device configs show up as config params. Can we make some change such that model and device configs don't show up in list of config args?

Reply:
it could be done by creating a class similar to basepolyconfig, for which we don't propagate the args